lib/matplotlib/pyplot.py

```diff
@@ -709,7 +709,7 @@ def xkcd(
     Turn on `xkcd <https://xkcd.com/>`_ sketch-style drawing mode.  This will
     only have effect on things drawn after this function is called.
 
-    For best results, the "Humor Sans" font should be installed: it is
+    For best results, the "xkcd script" font should be installed: it is
```

Comment:
```suggestion
    For best results, the `xkcd script <https://github.com/ipython/xkcd-font/>` font should be installed: it is
```
(although one may argue if one should point to the repo or the exact location, I think the repo makes more sense.)

Reply:
Ahh, OK! But maybe better to link to https://github.com/dummy-index/xkcd-font/tree/brushup then?